package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Sample nodejs client for the Everactive API",
   "main": "index.js",
   "scripts": {
-    "mhm-client": "node simple-mhm-client.js"
+    "mvm-client": "node simple-mhm-client.js",
```

Comment:
Could we update the file name as part of this change as well? Otherwise LGTM :+1: